### DISCUSSION TITLE
Extremely loud noise/level  75db when inserting plugin in AG chain

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#737] [Help]
Hi,

So sometimes a really weird things happen : adding  a new plugin in an AG insert chain produces extremly loud artifacts/noise and the levels become crary (+75db in my cubase vumeter) ! 
Levels and signal become crazy too in the inserted plugin (according to their meter or graph). This noise is constant.
I have encountered that with some Fabfilter and Waves plugins for example.
When that happens removing the plugin makes everything ok again but then you are stuck !
Test config is win10 server with catalina/cubase pro 11 client.

**Comment by apohl79** [comment]
Do you see any errors in the logs when this happens?
